boxxy/public/css/timeline.css

```diff
@@ -229,6 +229,8 @@ Modules - reusable parts of our design
   color: #303e49;
   margin-bottom: 0;
   margin-top: 0;
+  font-size: 20px;
```

Comment:
This is already the case for screens larger than 768px, as seen on line 275. It makes no sense to make the font size effectively larger when the screen size gets smaller.